bundles/lpar/jammy-zed-ovn-edge.yaml

```diff
@@ -0,0 +1,476 @@
+# Open Virtual Network (OVN) - requires Train or later
+#
+# NOTE: Please review the value for the configuration option
+#       `bridge-interface-mappings` for the `ovn-chassis` charm.
+#       Refer to the [Open Virtual Network (OVN)](https://docs.openstack.org/project-deploy-guide/charm-deployment-guide/latest/app-ovn.html)
+#       section of the [OpenStack Charms Deployment Guide](https://docs.openstack.org/project-deploy-guide/charm-deployment-guide/latest/)
+#       for more information.
+---
+local_overlay_enabled: False
+series: jammy
+variables:
+  openstack-origin:        &openstack-origin        cloud:jammy-zed
+  openstack-charm-channel: &openstack-charm-channel zed/edge
+  ovn-charm-channel:       &ovn-charm-channel       22.09/edge
```

Comment:
I don't see this channel available for ovn.

Reply:
it's empty at the moment, if you run `charmcraft status ovn-chassis` is there, but not on charmhub.io